WORKSPACE

```diff
@@ -1621,3 +1621,11 @@ go_repository(
     sum = "h1:oUQredbOIzWIMmeGR9dTLzSi4DqRVwxrPzSDiLJBp4Q=",
     version = "v0.0.0-20200310214500-3283b9706406",
 )
+
+go_repository(
+    name = "com_github_wealdtech_eth2_signer_api",
+    build_file_proto_mode = "disable_global",
+    importpath = "github.com/wealdtech/eth2-signer-api",
```

Comment:
this includes a ton of methods for admin and management that are irrelevant for the validator. I think maybe have the minimum for the remote wallet, anything else can be user side implemented.

Reply:
I don't see this being an issue, there is no requirement on a client to call all methods in the interface.

Reply:
Exactly, I think we should keep this spec as simple and minimalist as possible.

Reply:
eth2-signer-api defines the endpoints for the remote server, not the client.  The ability to unlock wallets and accounts (which I believe are the only endpoints unused by the validator) are not worth spinning out in to a separate repository; clients can just ignore the bits they don't use.